modules/installation-load-balancing-user-infra.adoc

```diff
@@ -44,12 +45,34 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-bare-metal"]
 :restricted:
 endif::[]
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp:
+endif::[]
 
 :_content-type: CONCEPT
 [id="installation-load-balancing-user-infra_{context}"]
 = Load balancing requirements for user-provisioned infrastructure
 
+ifndef::osp[]
 Before you install {product-title}, you must provision the API and application ingress load balancing infrastructure. In production scenarios, you can deploy the API and application ingress load balancers separately so that you can scale the load balancer infrastructure for each in isolation.
+endif::osp[]
+
+ifdef::osp[]
+[IMPORTANT]
+====
+[subs="attributes+"]
+Deployment on OpenStack with User-Managed Load Balancers is a Technology Preview feature only. Technology Preview features
```

Comment:
This is TechPreview for baremetal, nutanix, vsphere, ovirt and OSP. At least the code is there and is the same for all platforms.

Reply:
@EmilienM Who's doing QE for this feature? Are they testing all of those platforms? 

I can remove OpenStack-specific language from this and just keep generic language for on-prem platforms in this module, but I can't claim support for anything that isn't at least okayed by QE.

Reply:
Only OpenStack is doing QE on this one...
@rlobillo is the point of contact.

> I can remove OpenStack-specific language from this and just keep generic language for on-prem platforms in this module, but I can't claim support for anything that isn't at least okayed by QE.
Sounds good. it's TechPreview anyway (for all platforms that have the feature).